ci/build-linux.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+
+PYTHON_VERSION="${1}"
+UBUNTU_RELEASE=$(sudo lsb_release -rs)
+
+# Update APT repositories
+sudo apt-get update
+
+# Update pip
+python3 -m pip install --user --upgrade pip
+
+# Install vEnv
+python3 -m pip install --user virtualenv
+
+# Install curl
+sudo apt-get install -y curl libcurl4 libcurl4-gnutls-dev bc
+
+# Install Qt
+if [ "${UBUNTU_RELEASE}" = "22.04" ]; then
+  sudo apt-get install -y qtbase5-dev qt5-qmake qtbase5-dev-tools qttools5-dev-tools
+else
+  sudo apt-get install -y qt5-default qt5-qmake qtbase5-dev-tools qttools5-dev-tools
+fi
+
+# Create the Python vEnv
+python3 -m venv pext-env
+source pext-env/bin/activate
+python3 -m pip install --upgrade pip
+
+# Install Tox Travis
+python3 -m pip install tox-travis
```

Comment:
I think we can just use regular tox nowadays actually, as we moved from Travis to GitHub Actions quite some time ago.

Reply:
Solved in commit b738120.

GitHub's actions are green and the artifacts in place: https://github.com/airvzxf/Pext/actions/runs/3324391960.

Pext action: https://github.com/Pext/Pext/actions/runs/3324392289